plugins/azure_blob_storage/plugin.spec.yaml

```diff
@@ -423,7 +433,6 @@ actions:
           - PageBlob
           - AppendBlob
         example: BlockBlob
-        default: PageBlob
```

Comment:
and here, and the others

Reply:
Default overwrites the example field as @rmurray-r7 mentioned

Reply:
It overwrites the example field in the `help.md`, but I would keep the defaults in, especially for enums because they act as a placeholder option for users in the UI.

A workaround for now would be to make the examples match the defaults.